etc/telegraf.conf

```diff
@@ -736,6 +746,21 @@
 #   ## aggregator and will not get sent to the output plugins.
 #   drop_original = false
 
+ [[aggregators.metricaggregates]]
```

Comment:
adding config for metricaggregates 2 times creates 2 instances of the plgin with different periods.